### PR TITLE
fix(sounds): Prevent autopilot sound on Cold and Dark spawn

### PR DESCRIPTION
**Pull request by RichardPilbery**
Fixes #[issue_no]

## Summary of Changes

This PR prevents the autopilot disconnected sound playing when spawning into the sim.

Code from @Saschl 

## Additional context

Discord username (if different from GitHub):

## Testing instructions

1. Spawn in C+D
2.  Confirm autopilot disconnect sounds does not play
3. Undertake a flight
4. Confirm that the autopilot disconnect sounds plays when AP push button pressed or yoke is moved while AP is on

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Review by hotshotp — APPROVED** [review]
LGTM

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -246,6 +246,8 @@
 1. [FMGC] Implemented Airport Button - @patmack14 (Patrick Macken)
 1. [RMP] Fixed colour of SEL indicator - @tracernz (Mike)
 1. [MISC] Added custom Autobrake event for SimConnect - @aguther (Andreas Guther)
+1. [EWD] Fixed failures not re-triggering when resolved - @tricky_dicky (Richard Pilbery)
```
Comment (Saschl) [inline .github/CHANGELOG.md:249]: Shouldn't be here I guess? :)

**Review by Saschl — APPROVED** [review]

File: .github/CHANGELOG.md (hunk 1 above)
Comment (RichardPilbery) [reply .github/CHANGELOG.md:249]: Well, it got missed off the PR it relates too and has been merged, so the changelog needs updating...seemed a bit overkill to make it a separate PR.

Comment (Saschl) [reply .github/CHANGELOG.md:249]: Fair point^^

**Comment by Saschl** [comment]
Works perfectly fine. No sound when spawning C&D and runway, AP disconnect sound still playing when required (manual disconnect, "forced" disconnect

**Pull request merged**